### DISCUSSION TITLE
set border radius of label

### DISCUSSION THREAD

**Discussion by yanglei629** [HandyOrg/HandyControl#1195] [Q&A] [answered]
Hi,I do not want radius in label,how can i set it.

##snapshot
![image](https://user-images.githubusercontent.com/81857996/170690085-8d03ff1e-410c-4237-9104-51fb6490c6d5.png)

**Comment by Enat0** [comment, marked as answer]
Hello.

Try:
hc:BorderElement.CornerRadius="0"

**Comment by yanglei629** [comment]
@Enat0  Thanks, it is work.
